content/doc/developer/book.yml

```diff
@@ -15,5 +15,6 @@ chapters:
   - cli
   - testing
   - plugin-development
+  - blueocean-plugin-development
```

Comment:
Since this is still fairly new and not fully baked, I think this section should remain towards the rear until it's solidified and less "alpha"